apps/example-app/src/app/examples/22-signal-inputs.component.spec.ts

```diff
@@ -7,42 +7,45 @@ test('works with signal inputs', async () => {
     inputs: {
       ...aliasedInput('greeting', 'Hello'),
       name: 'world',
+      age: '4500000000',
     },
   });
 
   const inputValue = within(screen.getByTestId('input-value'));
-  expect(inputValue.getByText(/hello world/i)).toBeInTheDocument();
+  expect(inputValue.getByText(/hello world of 4500000000 years old/i)).toBeInTheDocument();
 });
 
 test('works with computed', async () => {
   await render(SignalInputComponent, {
     inputs: {
       ...aliasedInput('greeting', 'Hello'),
       name: 'world',
+      age: '4500000000',
     },
   });
 
   const computedValue = within(screen.getByTestId('computed-value'));
-  expect(computedValue.getByText(/hello world/i)).toBeInTheDocument();
+  expect(computedValue.getByText(/hello world of 4500000000 years old/i)).toBeInTheDocument();
 });
 
 test('can update signal inputs', async () => {
   const { fixture } = await render(SignalInputComponent, {
     inputs: {
       ...aliasedInput('greeting', 'Hello'),
       name: 'world',
+      age: '4500000000',
```

Comment:
To keep this a bit easier to read 😅

```suggestion
      age: '45',
```

Reply:
Modified ✅